compose/ui/ui/api/desktop/ui.api

```diff
@@ -3119,6 +3119,24 @@ public final class androidx/compose/ui/platform/LocalSoftwareKeyboardController
 	public final fun provides (Landroidx/compose/ui/platform/SoftwareKeyboardController;)Landroidx/compose/runtime/ProvidedValue;
 }
 
+public final class androidx/compose/ui/platform/PlatformInsets {
```

Comment:
There was a mistake on CI, and it didn't run the checks. I fixed it here:
https://teamcity.jetbrains.com/admin/settingsDiffView.html?id=buildType:JetBrainsPublicProjects_Compose_DesktopTestsForPullRequests&versionBefore=49&versionAfter=50&actionId=1756398

Now it run API checks properly.

I merge it know to fix the CI. If anything needs to be adressed, let's do that in the next PR's